src/sage/sets/disjoint_set.pyx

```diff
@@ -531,57 +525,56 @@ cdef class DisjointSet_of_integers(DisjointSet_class):
             sage: d = DisjointSet(5)
             sage: sorted(d.root_to_elements_dict().items())
             [(0, [0]), (1, [1]), (2, [2]), (3, [3]), (4, [4])]
-            sage: d.union(2,3)
+            sage: d.union(2, 3)
             sage: sorted(d.root_to_elements_dict().items())
             [(0, [0]), (1, [1]), (2, [2, 3]), (4, [4])]
-            sage: d.union(3,0)
+            sage: d.union(3, 0)
             sage: sorted(d.root_to_elements_dict().items())
             [(1, [1]), (2, [0, 2, 3]), (4, [4])]
             sage: d
             {{0, 2, 3}, {1}, {4}}
         """
-        s = {}
-        cdef int i
+        cdef dict s = {}
+        cdef int i, o
         for i in range(self.cardinality()):
             o = self.find(i)
```

Comment:
I thought it was faster to use `OP_find` :-)

Reply:
Indeed :O. I think these functions are not called as often as find and union.